sensor_msgs/Image normalisedImage
---
std_msgs/Int32 pixelOffset